README.md

```diff
@@ -1,75 +1,44 @@
-[<img align="right" src="https://img.shields.io/github/issues/mozillabrasil/sumo_live_helper.svg">](https://github.com/mozillabrasil/sumo_live_helper/issues)
-[<img align="right" src="https://img.shields.io/github/license/mozillabrasil/sumo_live_helper.svg">](https://github.com/mozillabrasil/sumo_live_helper/blob/master/LICENSE)
-[<img align="right" src="https://img.shields.io/github/forks/mozillabrasil/sumo_live_helper.svg">](https://github.com/mozillabrasil/sumo_live_helper/network/members)
-[<img align="right" src="https://img.shields.io/github/stars/mozillabrasil/sumo_live_helper.svg">](https://github.com/mozillabrasil/sumo_live_helper/stargazers)
-[<img align="right" src="https://img.shields.io/github/release/mozillabrasil/sumo_live_helper.svg">](https://github.com/mozillabrasil/sumo_live_helper/releases)
+[ ![Get Addon][amo-button] ][amo-website]
 
-![SUMO Live Helper](/src/res/logo/sumo64.png)
-# SUMO Live Helper [<img align="right" src="https://addons.cdn.mozilla.net/static/img/addons-buttons/AMO-button_2.png">](https://addons.mozilla.org/en-US/firefox/addon/sumo-live-helper-/)
-The goal of this Add-on is give to all volunteers of SUMO (Support Mozilla) a tool to see when there are new support questions in the forums that have not yet been answered. This will help to improve the Mozilla products user experience by getting answers to their problems more quickly.
+# ![SUMO Live Helper](src/res/logo/sumo32.png) SUMO Live Helper
 
-![Interface of SUMO Live Helper](sumo-live-helper.png)
+A WebExtension to help SUMO contributors to quickly and easily find unanswered 
+questions on support.mozilla.org forum.
 
-<!--
-![SUMO_Live_Helper gif](sumo_live_helper.gif)
--->
+![Interface of SUMO Live Helper](screenshot.png)
 
 ## Requirements
-* Firefox 62.0 or later
 
-## Features
-This Add-on shows how many questions are open and without answer on the support forum. By clicking the extension icon on the toolbar, a popup will open, allowing you to select an unanswered question. Selecting an unanswered question will open it in a new tab.
+* Operating system: Windows, MacOS, Linux and Android
+* Firefox version: ESR or greater
```

Comment:
It looks like you are missing the version number requirement.

Reply:
My goal was to remove the version number so we forgot to update the file. Maybe I could change it for "Latest ESR" so it's more precise. Would that by good for you?